src/components/Reviews.jsx

```diff
@@ -0,0 +1,39 @@
+import { useState, useEffect } from 'react'
+import { fetchReviews } from './Apis'
+
+const Reviews = () => {
+
+    const [reviews, setReviews] = useState([]);
+
+    useEffect(() => {
+        fetchReviews().then((response) => {
+            setReviews(response.reviews)
+        })
+    }, [])
+
+    console.log(reviews)
+
+    return (
+        <>
+            <p>Game Reviews</p>
+                <section id="reviews">
+                    {
+                        reviews.map((review) => {
```

Comment:
This is working fine because your default state is an empty array, but I might consider the isLoading pattern here, also what might happen if there is some kind of error. Whenever we do an api requests we probably want to consider some sort of error handling. :)

Reply:
isLoading pattern added in. 
Looking into error handling....

Reply:
Basic error handling for SQL errors now working with a lot of help from Jim & August